docs/source/modes.rst

```diff
@@ -78,10 +78,26 @@ It can happen that several buses influence each other while simultaneously charg
 In the end, the largest number of rotations that produce a non-negative result when taken together is returned as the optimized scenario.
 
+Recombination: split negative depot rotations into smaller rotations
+-------------
+::
+
+    mode = split_negative_depb
+
+This mode recombines trips from negative depot rotations into new rotations such that all new rotations are guaranteed to stay within their vehicle's capacity while leaving trips from the same rotation together as much as possible. This is done in two steps:
+First, the trips of each rotation are analysed. Trips from or to a depot are inserted into a look-up-table to be able to generate new depot-trips as needed.
+Second, new rotations are generated from the trips of negative depot rotations. The first trip (by time of departure) must be the start of a new rotation. In addition, the bus has to drive from a depot to the departure station. And finally, it must be able to drive back to a depot again without exceeding capacity. A simple rotation must therefore consist of three trips: from a depot to the departure station (Einsetzfahrt), the trip itself and from the arrival station back to a depot (Aussetzfahrt). Only if the overall consumption is less than the capacity of the bus can the rotation be considered valid at all. If that is not the case, this trip must be discarded.
+Side note: the trips to or from the depot might not be part of any original rotation. In this case, default values for distance and speed are assumed. These can be set in the configuration as default_depot_distance and default_mean_speed, respectively.
+After making sure the first trip is possible, the next trip of the same original rotation is checked. Now it must be possible to have the Einsetzfahrt, the first trip, the second trip and a new Aussetzfahrt (not necessarily to the same depot as before). Again, the consumption must not exceed the capacity. If that is not the case, the new rotation ends after the first trip. The second trip must be evaluated later and will be the start of a new rotation.
+In this fashion, all the trips of a rotation are checked if they can be made even without opportunity charging. Rotations will not become longer (different rotations are not mixed), but a smart vehicle disposition may assign the same vehicle to multiple rotations.
+Naming convention of new rotations: the new identifier will start with the original rotation name, followed by the letter "r" and may end with a counter of new rotations if there are multiple new recombinations from the same original, all parts separated by underscores. If the original rotations was named "Monday", the following recombination identifiers are possible (but not limited to this):
+- Monday_r (first recombination)
+- Monday_r_3 (third recombination of first leg)
```

Comment:
Would "Recombination of the [first/ third] part of the original rotation" also be a valid description?

Reply:
removed leg (eww...)